apps/journeys-admin/src/components/Editor/Toolbar/Toolbar.tsx

```diff
@@ -16,21 +16,31 @@ import logo from '../../../../public/taskbar-icon.svg'
 import { EDIT_TOOLBAR_HEIGHT } from '../constants'
 
 import { ActiveContent } from '@core/journeys/ui/EditorProvider'
+import { setBeaconPageViewed } from '@core/journeys/ui/setBeaconPageViewed'
 import Button from '@mui/material/Button'
+import { useRouter } from 'next/router'
 import { useTranslation } from 'react-i18next'
 import { Items } from './Items'
 import { Menu } from './Menu'
+import { TitleDescriptionDialog } from './TitleDescriptionDialog'
 
 export function Toolbar(): ReactElement {
+  const router = useRouter()
+  const { t } = useTranslation('apps-journeys-admin')
   const { journey } = useJourney()
-  const { t } = useTranslation('journeys-admin')
+  const [titleDialogOpen, setTitleDialogOpen] = useState<boolean>(false)
```

Comment:
These can be a bit more generic in naming since its the only dialog on the component
```suggestion
  const [dialogOpen, setDialogOpen] = useState(false)
```